src/components/podcast/Podcast.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Episode from './episode/Episode';
+import PodcastHeader from './Header'
+import parsePodcastFeedFromXML from '../../util/podcast-util';
+
+/**
+ * Component for displaying the details about a podcast including the episodes
+ */
+const Podcast = (props) => {
+
+    // Track the podcast object
+    const [podcast, setPodcast] = useState({});
+    // Track the episodes separately
+    const [episodes, setEpisodes] = useState([])
+    // Track if the podcast has been loaded
+    const [loaded, setLoaded] = useState(false);
+
+    useEffect(() => { loadFeed(props.feed + '.xml') });
+
+    /**
+     * If the feed hasn't been loaded go out and load it.  Set the podcast
+     * state and loaded to true.
+     */
+    async function loadFeed(feedFile) {
+        if (!loaded) {
+            axios.get(feedFile)
+                .then(function (result) {
+
+
+                    const podcast = parsePodcastFeedFromXML(result.data);
+                    setPodcast(podcast);
+                    setEpisodes(podcast.episodes);
+                    setLoaded(true);
+                    if (podcast) {
+                        document.title = podcast.title;
+                    }
+                })
+        }
+    }
+
+    /** 
+     * Handle the click of the Order Flip button, which will reverse
+     * the episode list and update the state
+    */
+    const handleOrderFlip = () => {
+        episodes.reverse();
+        setEpisodes([...episodes]);
+    }
+
+
+    return (
+        <div><header><PodcastHeader podcast={podcast} /></header>
+            {episodes && episodes.length > 1 ? <button type="button" onClick={handleOrderFlip} className='flip-button'>Flip Order</button> : null}
+            <ul>
+                {episodes.map((value, index) => {
+                    return <div key={index}><Episode episode={value} /></div>
```

Comment:
any reason you're wrapping in a div instead of just this?:

```
<Episode key={index} episode={value} />
```

Reply:
... then again, that div should actually be a `<li>` since you're in a list here.

Reply:
...also, this is another place where I really think implicit returns are cleaner